### ISSUE TITLE
"Editable" attribute has been reversed for person entities (created via config.yaml vs UI)

### ISSUE THREAD

**Issue by jamieshaw** [home-assistant/frontend#8900]
### The problem

Just fast forwarding through the last few months of Home Assistant updates (previously stable at 2021.1) — pausing for a few days on each.

Long story short — it seems the `editable` attribute for `person` entities has the incorrect boolean value for persons created via the **frontend** versus those created via **configuration.yaml**.  In previous instances, those from **config.yaml** would have `editable` set to `false` (cannot be edited from the FE); for UI-created users, this would be `true`.

This is now reversed in 2021.3 — **config.yaml** persons are showing `editable: true`.

---

_**Note**: This may be fixed in 2021.4 but isn't mentioned in the change notes. I haven't had chance to upgrade yet, finding time to patch breaking changes before updating fully_

### What is version of Home Assistant Core has the issue?

core-2021.3.4

### What was the last working version of Home Assistant Core?

core-2021.2.3

### What type of installation are you running?

Home Assistant Core

### Integration causing the issue

person

### Link to integration documentation on our website

https://www.home-assistant.io/integrations/person/

### Anything in the logs that might be useful for us?

```txt
No errors — just create two users, one UI based, one config.yaml based.
```

**Comment by jamieshaw** [comment]
Having now updated to 2021.4.5, I can confirm the issue persists.

**Comment by jamieshaw** [comment]
Issue continues to persist as of Home Assistant 2021.5.0

**Comment by jamieshaw** [comment]
Issue continues to persist as of Home Assistant 2021.6.0

**Comment by bramkragten** [comment]
Can you please stop adding a comment each version bump? If it is looked at we will let you now in the issue, you can respond when the Github bot asks you to.

**Comment by jamieshaw** [comment]
Apologies; but just keeping abreast of the issue — I know in the past, minor issues such as this have fallen stale.

Now that's actually seen _some_ activity (by way of your comment), I'll hang fire 👍🏻

**Comment by spacegaier** [comment]
Do you see any impact of the wrong attribute (which I assume is coming like that from the backend)? In my instance I can still only edit the ones NOT coming from YAML.

**Comment by spacegaier** [comment]
I think this is a backend topic. Paulus changed the logic in PR https://github.com/home-assistant/core/pull/45528 that sets this `editable` value.

**Comment by jamieshaw** [comment]
> I think this is a backend topic. Paulus changed the logic in PR https://github.com/home-assistant/core/pull/45528 that sets this `editable` value.

I agree this is a BE issue – it was created in core but transferred to frontend; assuming as a result of including notion of users created _via_ the frontend.

Probably _very_ instance specific to me personally, but this was a breaking change for me. My wife and I are hard coded into the YAML (`editable: false`) with guests, who are tracked by MAC address, created via the frontend (`editable: true`) – I use this attribute to filter users/guests appropriately. This determines how certain scripts/automations run.

Right now; I've just flipped all Filters that check this currently.

Semantically though – the value is incorrect. As YAML based entities are not `editable`.

**Comment by spacegaier** [comment]
So based on the discussion this is not a frontend topic, since it behaves correctly here, but rather something seems to have changed in the backend, although on first glance the linked core PR seems fine to me (https://github.com/home-assistant/core/blob/27764e998570d9c8224b61226fc7d2ed96ee24b2/homeassistant/components/person/__init__.py#L375), but I did not debug the backend.
